vsc_pdf_template_transformer/lib/models/tpl_sized_box.dart

```diff
@@ -29,9 +34,28 @@ class TplSizedBox implements wb.WidgetBuilder {
   @override
   Widget? buildWidget(Map<String, dynamic> data) {
     var value = SizedBox(
-      width: width,
-      height: height,
-    );
+        width: _evaluateInput(width, data),
+        height: _evaluateInput(height, data),
+        child: child?.buildWidget(data));
     return value;
   }
+
+  double? _evaluateInput(String? input, Map<String, dynamic> data) {
```

Comment:
I think you're going to find that this method will get almost entirely repeated for every widget. If we're going to with `TplString` for all properties, why not move it to `String? tplString.evaluateValue()`? Then you only have have to ask the TplString for the evaluated value and parse it to a double here.

Which brings me back to `TplDouble`... If you had a TplDouble type, you could have a corresponding method `double? tplDouble.evaluateValue()` which would take care of the evaluation and parsing to a double.